cidc_schemas/prism.py

```diff
@@ -845,7 +845,8 @@ def merge_artifact(
     assay_type: str,
     file_size_bytes: int,
     uploaded_timestamp: str,
-    md5_hash: str,
+    crc32c_hash: str = None,
+    md5_hash: str = None,
```

Comment:
nit: shouldn't that be something like this then? 
```suggestion
    md5_hash: Union[str,None] = None,
```